.gitsubprojects

```diff
@@ -2,4 +2,4 @@
 git_subproject(Brion https://github.com/BlueBrain/Brion.git 015515f)
 git_subproject(vmmlib https://github.com/Eyescale/vmmlib.git 2bec113)
 git_subproject(Lunchbox https://github.com/Eyescale/Lunchbox.git f48795e)
```

Comment:
We updated them in all other projects to the released versions - in any case you'll get new ones.